src/documentation/models/models.ts

```diff
@@ -78,7 +78,14 @@ export interface ITableColumn {
   width: string;
 }
 
+interface ITableContentValue {
+  code: string;
+  text: string;
+  code2?: string;
+  text2?: string;
+}
+
```

Comment:
Can we achieve this with `code: string[]` and `text: string[]`? Make the necessary changes in all the components that use this interface

```suggestion
export interface ITableContentValue {
  code: string[];
  text: string[];
}

```